charts/tensorleap/templates/rabbitmq-cm.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: rabbitmq-config
+data:
+  enabled_plugins: |
+    [rabbitmq_management].
+  rabbitmq.conf: |
+    cluster_formation.k8s.host = kubernetes.default.svc.cluster.local
+    log.file.level = debug
+    log.console = true
+    log.console.level = debug
+    cluster_formation.k8s.address_type = hostname
+    cluster_formation.node_cleanup.interval = 30
+    cluster_formation.node_cleanup.only_log_warning = true
+    cluster_partition_handling = autoheal
+    queue_master_locator=min-masters
+    loopback_users.guest = false
```

Comment:
is all of this a must? whats wrong with the default?

Reply:
I think that the logs should stay (for debugging), I'll try removing the rest